### PR TITLE
Update license in README.md. Refresh from dev site

### LINKED ISSUE #19: License ambiguity
While README.md mentions Apache license 2.0, LICENSE file and code mentions Apache license 1.1. I think a single version should be used everywhere.

### PR DESCRIPTION
**Pull request by oschwald**
Closes #19.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +37,11 @@ Overview
 
 The mod_geoip2 module uses the libGeoIP library to look up geolocation
 information for a client as part of the http request process. This
-module is free software, and is licensed under the [Apache
-license](http://www.apache.org/licenses/LICENSE-2.0.html).
+module is free software, and is licensed under the [pache Software
```
Comment (oalders) [inline README.md]: s/pache/Apache/

File: README.md [hunk 2]
```diff
@@ -171,8 +173,8 @@ mod_remoteip provides.
     GeoIPScanProxyHeaderField FieldName
 
 Sometimes it is useful to use another field as the source for the
-client's IP address. You can set this directive to tell this module
-which header to look at in order to determine the client's IP address.
+client’s IP address. You can set this directive to tell this module
```
Comment (oalders) [inline README.md]: Just wondering why we're switching from `'` to `’`.

Comment (oschwald) [inline README.md]: Pandoc did this.

File: README.md (hunk 1 above)
Comment (oschwald) [inline README.md]: Doh, thanks.

**Pull request merged**